public/javascripts/SVLabel/src/SVLabel/canvas/ContextMenu.js

```diff
@@ -472,15 +472,38 @@ function ContextMenu (uiContextMenu) {
             _setTags(targetLabel);
             _setTagColor(targetLabel);
             if (getStatus('disableTagging')) { disableTagging(); }
+
+            // Hide the severity menu for the Pedestrian Signal label type.
+            if (labelType === 'Signal') {
+                $severityMenu.css({visibility: 'hidden', height: '0px'});
+            } else {
+                $severityMenu.css({visibility: 'inherit', height: '50px'});
+            }
             var windowHeight = $menuWindow.outerHeight();
+            var connectorBuffer = 3; // for connector to overlap border of label icon
```

Comment:
It looks like this is a constant, maybe change this to be called `CONNECTOR_BUFFER` and move it to the top of the file, maybe right after the section where `$connector` is defined? Somewhere around line 27.